<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDb2NewConnectionBase</class>
 <widget class="QDialog" name="QgsDb2NewConnectionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a New DB2 connection</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>440</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>671</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection Properties</string>
   </property>
   <widget class="QLabel" name="DB2HostLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DB2 Host:</string>
    </property>
   </widget>
   <widget class="QLabel" name="DB2port">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>DB2 Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="database">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Database:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPort">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDatabase">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHost">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Test connection</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtService">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DB2ServiceLabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Service/DSN:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDriver">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DB2ServiceLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Driver:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DB2ServiceLabel_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection Name:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabAuthentication">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>417</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabBasic">
     <attribute name="title">
      <string>Authentication</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel3">
        <property name="text">
         <string>Username</string>
        </property>
        <property name="buddy">
         <cstring>txtUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="chkStoreUsername">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel3_2">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="buddy">
         <cstring>txtPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtUsername"/>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="chkStorePassword">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="db2ConnectStatus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DB2 Connect Status: </string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtService</tabstop>
  <tabstop>txtDriver</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>txtDatabase</tabstop>
  <tabstop>tabAuthentication</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>chkStoreUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>chkStorePassword</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnConnect</sender>
   <signal>released()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>on_btnConnect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
